extensions/big-data-cluster/src/installer/installer.ts

```diff
@@ -29,7 +32,7 @@ export async function installKubectl(shell: Shell): Promise<Errorable<null>> {
     const downloadFile = path.join(installFolder, binFile);
     const downloadResult = await download.to(kubectlUrl, downloadFile);
     if (failed(downloadResult)) {
-        return { succeeded: false, error: [`Failed to download kubectl: ${downloadResult.error[0]}`] };
+        return { succeeded: false, error: [localize('downloadKubectlFailed', 'Failed to download kubectl: ${0}', downloadResult.error[0])] };
```

Comment:
I think you have to remove the dollor sign here. previous it was using template string but not it is not required